### DISCUSSION TITLE
ElDropdown Add multi-level menu functionality

### DISCUSSION THREAD

**Discussion by liu-kuan** [element-plus/element-plus#16109] [Feature Request]
下拉菜单组件可以新增一下多级菜单的功能，像下图这样的

<img width="420" alt="企业微信截图_d7496352-cac0-45da-a206-e66928c10560" src="https://github.com/element-plus/element-plus/assets/74317843/d50852bc-ac6a-4c74-a327-5fe5fc7fec4a">

**Comment by shungdawei** [comment]
这么常见的场景居然不支持，都几年时间了
